siriuspy/siriuspy/currinfo/lifetime/main.py

```diff
@@ -0,0 +1,166 @@
+"""Main Module of the IOC Logic."""
+
+import warnings
+import time as _time
+from collections import deque as _deque
+import numpy as _np
+from epics import PV as _PV
+
+from siriuspy.callbacks import Callback as _Callback
```

Comment:
@anacso17 , we are trying to standardize imports of modules in the same package using relative paths, like:

```Python
from ...callbacks import Callback as _Callback
from ...epics import SiriusPVTimeSerie as _SiriusPVTimeSerie
from ...envars import VACA_PREFIX as _vaca_prefix

from ..csdev import \
    Const as _Const, get_lifetime_database as _get_database
```

what do you think? It is not clear to me whether relative is preferable over absolute paths. I find absolute easier to read but I noticed that trend among us the trend was to use relative paths.

Reply:
I also find absolute easier to read, but I will follow the trend. I will commit the changes soon